client/my-sites/customer-home/cards/tasks/site-setup-list/index.jsx

```diff
@@ -215,94 +214,71 @@ const SiteSetupList = ( {
-			{ ( ! useDrillLayout || currentDrillLayoutView === 'task' ) && (
-				<div className="site-setup-list__task task">
-					<div className="site-setup-list__task-text task__text">
-						{ currentTask.isCompleted ? (
-							<Badge type="info" className="site-setup-list__task-badge task__badge">
-								{ translate( 'Complete' ) }
-							</Badge>
-						) : (
-							<div className="site-setup-list__task-timing task__timing">
-								<Gridicon icon="time" size={ 18 } />
-								{ translate( '%d minute', '%d minutes', {
-									count: currentTask.timing,
-									args: [ currentTask.timing ],
-								} ) }
+
+			<div className="site-setup-list__nav">
+				{ ! useDrillLayout && <CardHeading>{ translate( 'Site setup' ) }</CardHeading> }
+				{ tasks.map( ( task ) => {
+					const enhancedTask = getTask( task );
+					const isCurrent = task.id === currentTask.id;
+					const isCompleted = task.isCompleted;
+
+					return useDrillLayout && isCurrent ? (
+						<div
+							className={ classnames( 'nav-item', {
+								'is-current': isCurrent,
+							} ) }
+						>
+							<div className="nav-item__status">
+								{ isCompleted ? (
+									<Gridicon className="nav-item__complete" icon="checkmark" size={ 18 } />
+								) : (
+									<div className="nav-item__pending" />
+								) }
 							</div>
-						) }
-						<h2 className="site-setup-list__task-title task__title">{ currentTask.title }</h2>
-						<p className="site-setup-list__task-description task__description">
-							{ currentTask.description }
-						</p>
-						<div className="site-setup-list__task-actions task__actions">
-							{ currentTask.actionText && (
-								<Button
-									className="site-setup-list__task-action task__action"
-									primary
-									onClick={ () =>
-										startTask( dispatch, currentTask, siteId, advanceToNextIncompleteTask )
-									}
-									disabled={
-										currentTask.isDisabled ||
-										( currentTask.isCompleted && currentTask.actionDisableOnComplete )
-									}
-								>
-									{ currentTask.actionText }
-								</Button>
-							) }
-							{ currentTask.isSkippable && ! currentTask.isCompleted && (
-								<Button
-									className="site-setup-list__task-skip task__skip is-link"
-									onClick={ () => {
-										setTaskIsManuallySelected( false );
-										skipTask( dispatch, currentTask, tasks, siteId, setIsLoading );
-									} }
-								>
-									{ translate( 'Skip for now' ) }
-								</Button>
-							) }
+							<CurrentTaskItem
+								currentTask={ currentTask }
+								skipTask={ () => skipTask( dispatch, currentTask, tasks, siteId, setIsLoading ) }
+								startTask={ () =>
+									startTask( dispatch, currentTask, siteId, advanceToNextIncompleteTask )
+								}
+								setTaskIsManuallySelected={ setTaskIsManuallySelected }
+							/>
+							{ useDrillLayout &&
+								( isCurrent ? (
+									<Gridicon className="nav-item__chevron" icon="chevron-up" size={ 18 } />
```

Comment:
Since the icon is the only conditional piece here, can we simplify this to `<Gridicon className="nav-item__chevron" icon={ isCurrent ? 'chevron-up' : 'chevron-down'} size="18" />`?